config/nginx.conf

```diff
@@ -1,6 +1,12 @@
 # This is the version used in development environments
 server {
     listen 8079;
+    server_name micromasters.herokuapp.com;
+    return 301 https://micromasters.mit.edu$request_uri;
+}
+
+server {
+    listen 8079 default_server;
```

Comment:
I'm not sure these changes are necessary in this file since this file is only for dev environments